extension/js/agent/utils/serializeObject.js

```diff
@@ -90,13 +90,28 @@
       return this.classPropertyCache[info.name];
     }
 
-    var props = this.serializeObject(_.pick(serializeObject, info.keys));
```

Comment:
Mind getting a test in there. I think we have coverage for that function

https://github.com/marionettejs/marionette.inspector/blob/master/extension/js/test/unit/agent/utils/serializeObject.spec.js